test/suppose-process.test.js

```diff
@@ -33,92 +34,66 @@ describe('process', function()
-      assert.strictEqual(packageObj.author, 'JP Richardson')
-      assert.strictEqual(packageObj.license, 'MIT')
 
       done()
     })
   })
 
-  it('should automate the command: npm init and output stdio', function(done)
+  it('should automate a prompt script and output stdio', function(done)
   {
     var self = this
 
     var debugFile = join(this.TEST_DIR, 'debug.txt')
     process.chdir(this.TEST_DIR)
 
-    suppose('npm', ['init'], {debug: fs.createWriteStream(debugFile)})
-      .when(/name\: \((\w|\-)+\)\s*/).respond("awesome_package\n")
-      //.when(/name: \(test-suppose\) /).respond('awesome_package\n')
-      .when(/version: \(1\.0\.0\) /).respond('0.0.1\n')
-      .when(/description: /).respond("It's an awesome package man!\n")
-      .when(/entry point: \(index\.js\) /).respond("\n")
-      .when(/test command: /).respond('npm test\n')
-      .when(/git repository: /).respond("\n")
-      .when(/keywords: /).respond('awesome, cool\n')
-      .when(/author: /).respond('JP Richardson\n')
-      .when(/license: \(ISC\) /).respond('MIT\n')
-      .when(/ok\? \(yes\) / ).respond('yes\n')
+    var options = {debug: fs.createWriteStream(debugFile), stripAnsi: true}
+
+    suppose('node', [`${__dirname}/helpers/cli-prompt.js`], options)
+      .when('name (mocked name)').respond('awesome_package\n')
+      .when('version (1.0.0)').respond('0.0.1\n')
     .end(function(code, signal)
     {
       assert.strictEqual(code, 0)
 
-      var packageObj = require(join(self.TEST_DIR, 'package.json'))
+      var packageObj = require(join(self.TEST_DIR, 'temp.json'))
 
       assert.strictEqual(packageObj.name, 'awesome_package')
       assert.strictEqual(packageObj.version, '0.0.1')
-      assert.strictEqual(packageObj.description, "It's an awesome package man!")
-      assert.strictEqual(packageObj.main, 'index.js')
-      assert.strictEqual(packageObj.scripts.test, 'npm test')
-      assert.strictEqual(packageObj.keywords[0], 'awesome')
-      assert.strictEqual(packageObj.keywords[1], 'cool')
-      assert.strictEqual(packageObj.author, 'JP Richardson')
-      assert.strictEqual(packageObj.license, 'MIT')
 
       var debugResFile = ''
       if (os.platform() === 'darwin')
-        debugResFile = P('test/resources/debug-darwin.txt')
+        debugResFile = `${__dirname}/resources/debug-darwin.txt`
       else
-        debugResFile = P('test/resources/debug-linux.txt')
+        debugResFile = `${__dirname}/resources/debug-linux.txt`
 
       //check debug file
       var debugString = fs.readFileSync(debugFile).toString()
                           .replace(/\r\n|\r|\n/g, '\n')
       var checkString = fs.readFileSync(debugResFile).toString()
                           .replace(/\r\n|\r|\n/g, '\n')
 
+      //remove first two lines as they contain platform-specific paths
+      debugString = debugString.split('\n').slice(2).join('\n')
+      checkString = checkString.split('\n').slice(2).join('\n')
+
```

Comment:
This should not be needed to match the output. Are you enabling the `debug` option? You should not use it for that...